Clarify support of PersistentVolume access modes (#148)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_preparing-your-openshift-environment-for-stf.adoc

[id="persistent-volumes"]
== Persistent volumes

> {ProjectShort} uses persistent storage in {OpenShiftShort} to instantiate the volumes dynamically so that Prometheus and ElasticSearch can store metrics and events. When persistent storage is enabled via the Service Telemetry Operator, the Persistent Volume Claims requested in an {ProjectShort} deployment should result in an access mode of RWO (ReadWriteOnce). If your environment is made up of pre-provisioned persistent volumes, be sure that volumes of RWO are available in the {OpenShiftShort} default configured `storageClass`. For more information about recommended configurable storage technology in {OpenShift}, see https://docs.openshift.com/container-platform/{SupportedOpenShiftVersion}/scalability_and_performance/optimizing-storage.html#recommended-configurable-storage-technology_persistent-storage[Recommended configurable storage technology].

Reviewer: ```suggestion
{ProjectShort} uses persistent storage in {OpenShiftShort} to instantiate the volumes dynamically so that Prometheus and ElasticSearch can store metrics and events. When persistent storage is enabled through the Service Telemetry Operator, the Persistent Volume Claims requested in an {ProjectShort} deployment results in an access mode of RWO (ReadWriteOnce). If your environment contains pre-provisioned persistent volumes, ensure that volumes of RWO are available in the {OpenShiftShort} default configured `storageClass`. For more information about recommended configurable storage technology in {OpenShift}, see https://docs.openshift.com/container-platform/{SupportedOpenShiftVersion}/scalability_and_performance/optimizing-storage.html#recommended-configurable-storage-technology_persistent-storage[Recommended configurable storage technology].
```

Author: This is great, thank you!
